website/content/docs/secrets/aws.mdx

```diff
@@ -305,6 +301,60 @@ boundary policies that you wish to ensure that Vault uses. This policy will
 ensure that Vault uses one of the permissions boundaries specified (not all of
 them).
 
+### Plugin Workload Identity Federation (WIF)
```

Comment:
Is this intentionally a subsection of "IAM permissions policy for Vault"? It seems like it shouldn't be.

Reply:
Good catch! Not intentional. Moved in https://github.com/hashicorp/vault/pull/25706/commits/a4f5a24251cfbe16cd566ee906d549ba826208aa.